tools/extra/plot.py

```diff
@@ -0,0 +1,117 @@
+# In the name of GOD the most compassionate the most merciful

```

Comment:
we Muslims, like to start everything in the name of God, in order to get better results.which is strange in the non-muslim cultures, as it is unprecedented. but for us, its considered a good habit to have, though not compulsory or anything. 
I myself tend to use that sentence to signify to myself that all I have is from the mercy of dear Lord and all I achieve is simply because of his graciousness and always seek his help to get better.